lock-impl/src/main/java/com/palantir/lock/impl/LegacyTimelockService.java

```diff
@@ -42,13 +48,20 @@
 import java.util.SortedMap;
 import java.util.TreeMap;
 import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+import java.util.function.LongSupplier;
 import java.util.stream.Collectors;
+import javax.annotation.Nullable;
+import one.util.streamex.EntryStream;
+import one.util.streamex.StreamEx;
 
 /**
  * A {@link TimelockService} implementation that delegates to a {@link LockService} and {@link TimestampService}.
  */
 public class LegacyTimelockService implements TimelockService {
 
+    private static final ClientId TIMESTAMP_LEASES_CLIENT_ID = ClientId.of("timestamp-leases");
```

Comment:
Should we make this a bit more cryptic or add some prefix, `nmtl::` like you recommended previously

I wouldn't be surprised if someone thinks to implement timestamp leases on top of timelock and names them similarly

Reply:
I wouldn't worry about this: *I hold* this implementation as short term, and we switch tests to timelock.